packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -105,6 +105,9 @@ class RunCommand extends RunCommandBase {
     argParser.addFlag('preview-dart-2',
         hide: !verboseHelp,
         help: 'Preview Dart 2.0 functionality.');
+    argParser.addFlag('track-widget-creation-locations',
+        hide: !verboseHelp,
+        help: 'Track widget creation locations. Requires Dart 2.0 functionality.');
```

Comment:
Would it make sense to explicitly state and check that this flag only works when `--preview-dart-2` flag is provided?

Reply:
done. Added a check for that like the newly added check for strong.